test/v1/configuration.go

```diff
@@ -65,18 +66,24 @@ func WaitForConfigLatestRevision(clients *test.Clients, names test.ResourceNames
 	err := WaitForConfigurationState(clients.ServingClient, names.Config, func(c *v1.Configuration) (bool, error) {
 		if c.Status.LatestCreatedRevisionName != names.Revision {
 			revisionName = c.Status.LatestCreatedRevisionName
+			// We also check that the revision is pinned, meaning it's not a stale revision.
+			// Without this it might happen that the latest created revision is later overriden by a newer one
+			// and the following check for LatestReadyRevisionName would fail.
+			if revErr := CheckRevisionState(clients.ServingClient, revisionName, IsRevisionPinned); revErr != nil {
+				return false, nil
+			}
 			return true, nil
 		}
 		return false, nil
 	}, "ConfigurationUpdatedWithRevision")
 	if err != nil {
-		return "", err
+		return "", errors.Wrapf(err, "LatestCreatedRevisionName not updated")
```

Comment:
Please do not use `github.com/pkg/errors` any longer. It's been replaced in go 1.13 with `fmt.Errorf` containing a `%w`.

In this case:

```suggestion
		return "", fmt.Errorf("LatestCreatedRevisionName not updated: %w", err)
```

(applicable everywhere)

Reply:
ok. Thanks